packages/contracts/contracts/test/Encoding.t.sol

```diff
@@ -56,6 +56,7 @@ contract Encoding_Test is CommonTest {
         assertEq(encoding, _encoding);
     }
 
+    // [Kroma: START]
```

Comment:
nit: wouldn't it be better to lower this to `// assert DepositTx part` (L#81)?

Reply:
I marked it at the function level, like `Hashing.t.sol`.